api/tests/unit/domain/usecases/authenticate-pole-emploi-user_test.js

```diff
@@ -189,26 +201,40 @@ describe('Unit | UseCase | authenticate-pole-emploi-user', function() {
       });
 
       // then
-      expect(tokenService.createAccessTokenFromUser).to.have.been.calledWith(userId, 'pole_emploi_connect');
+      expect(tokenService.createAccessTokenFromUser).to.have.been.calledWith(1, 'pole_emploi_connect');
     });
 
     it('should return accessToken and idToken', async function() {
       // given
+      const pixAccessToken = 'pixAccessToken';
+      const poleEmploiTokens = new PoleEmploiTokens({
+        accessToken: 'poleEmploiAccessToken',
+        expiresIn: 60,
+        idToken: 'idToken',
+        refreshToken: 'refreshToken',
+      });
       const expectedResult = {
         pixAccessToken,
         poleEmploiTokens,
       };
+      const userInfo = {
+        family_name: 'Morris',
+        given_name: 'Tuck',
+        externalIdentityId: '094b83ac-2e20-4aa8-b438-0bc91748e4a6',
+      };
 
+      authenticationService.getPoleEmploiUserInfo.resolves(userInfo);
+      authenticationService.generatePoleEmploiTokens.resolves(poleEmploiTokens);
```

Comment:
**Suggestion :** `withArgs` ?

Reply:
Pour le `generatePoleEmploiTokens` c'est testé explicitement dans le `should call authenticate pole emploi user with code, redirectUri and clientId parameters`.
Pour le `getPoleEmploiUserInfo` c'est testé aussi explicitement dans le `should call get pole emploi user info with id token parameter`.
Du coup le `withArgs` est déjà testé au moins une fois, pas besoin de le rajouter. 
Bon après dans mon idéal de tests, on aurait eu qu'un seul test (au lieu de deux voir plus) qui vérifie ces withArgs d'un seul coup. Mais comme les tests étaient déjà existant j'ai laissé tel quel x)

Reply:
ok ! 🙂